test/cmdlineTests/import_asm_json_optimize/output

```diff
@@ -0,0 +1,2 @@
+Opcodes:
+PUSH0 PUSH0 REVERT 
```

Comment:
I was just about to write that CLI optimizes the same input differently than Standard JSON, but it's just that there we have the text assembly output and here zeros are shown as `PUSH0` opcodes.

Perhaps would be better to request `--asm` here too. `--opcodes` is deprecated and will be removed eventually.

Reply:
Huh, I did not realize opcodes was deprecated. Is there a message/documentation of this somewhere that I missed?

Reply:
That's the plan: #14033.

Reply:
Going back to this, though, if it's planned to be removed for 0.9 (at least according to the associated milestone), shouldn't we issue some warning/deprecation messages somewhere, somehow?

Reply:
Yeah, we should. Not sure why we didn't do that while going over the milestone. Maybe we just didn't stop on this one. I mentioned it in https://github.com/ethereum/solidity/issues/15795#issuecomment-2656471399 now.